lib/instrumentation/pg.js

```diff
@@ -19,6 +19,7 @@ function getQuery(shim, original, name, args) {
   return statement
 }
 
+// eslint-disable-next-line sonarjs/cognitive-complexity
```

Comment:
we try not to add these overrides, they only exist for when we introduce the rule and are trying to pay down. i also noticed it exists in openai and i missed that.  I'll file a ticket to address

Reply:
:-/ and doing it again seems to have worked. I must've had an autocompletion error in my first attempt.